lib/isGurmukhi.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Checks if first char in string is part of the Gurmukhi Unicode block.
+ * @param {String} text The text to check.
+ * @return {boolean} True if Unicode Gurmukhi, false if other.
+ * @example
+ * isGurmukhi('ਗੁਰਮੁਖੀ') // => true
+ * isGurmukhi('gurmuKI') // => false
+ */
+const isGurmukhi = text => ( text.charCodeAt( 0 ) >= 2560 && text.charCodeAt( 0 ) <= 2687 )
```

Comment:
Is testing the first char sufficient? Seems a little cheeky lol. I guess it's sufficient for most cases, but maybe add a second parameter, `exhaustive [bool]` which will check every character in the string if `true`.

I do think that checking every character might be fine though, perf-wise, unless there's some mega long strings being passed in.

Reply:
That can be done.